unittests/resource_limits_test.cpp

```diff
@@ -4,19 +4,21 @@
 #include <eosio/chain/resource_limits.hpp>
 #include <eosio/chain/config.hpp>
 #include <eosio/testing/chainbase_fixture.hpp>
+#include <eosio/testing/tester.hpp>
 
 #include <boost/test/unit_test.hpp>
 
 using namespace eosio::chain::resource_limits;
 using namespace eosio::testing;
 using namespace eosio::chain;
 
-class resource_limits_fixture: private chainbase_fixture<1024*1024>, public resource_limits_manager
+class resource_limits_fixture: private tester, private chainbase_fixture<1024*1024>, public resource_limits_manager
 {
    public:
       resource_limits_fixture()
-      :chainbase_fixture()
-      ,resource_limits_manager(*chainbase_fixture::_db)
+      :tester()
+      ,chainbase_fixture()
+      ,resource_limits_manager(*tester::control, *chainbase_fixture::_db)
```

Comment:
Yeah no problem.

Reply:
Since the logger instance was later in the lifecycle via the `enable_deep_mind` call, I had to pass to a lambda that knows how to retrieve the deepmind logger instance.

I'll push what I have so far, missing piece will be the RAM event context object.